src/api/factories/EscrowLockFactory.ts

```diff
@@ -0,0 +1,39 @@
+import { inject, named } from 'inversify';
```

Comment:
can you create one EscrowFactory?
get(data: EscrowMessageInterface): Promise<Escrow> {

then based on the data, return Escrow